01-python/infrastructure/tree.py

```diff
@@ -1,3 +1,21 @@
-def draw_tree():
-    print("tree not found")
+def draw_tree(num=2):
+    printATree(num)
     return
+
+def printATree(num):
+  for _ in range(num):
```

Comment:
better to have the for loop in the parent function
```suggestion
  for _ in range(num):
      printATree()
```